src/Client/Web/Bit.Client.Web.BlazorUI/Components/Lists/BitBasicList.razor

```diff
@@ -0,0 +1,46 @@
+﻿@namespace Bit.Client.Web.BlazorUI
+@inherits BitComponentBase
+@typeparam TItem
+
+<div @ref="RootElement"
+     style="@StyleBuilder.Value"
+     class="@ClassBuilder.Value"
+     role="@Role">
+
+    @if (Virtualize)
+    {
+        <Virtualize Items="@Items"
+                    Context="item" 
+                    ItemSize="@ItemSize" 
+                    OverscanCount="@OverscanCount" >
+
+            <ItemContent >
+                @RowTemplate(item)
+            </ItemContent>
+
+            <Placeholder>
+                @if (LoadingText.ToString().Length > 0)
+                    {
+                    <p class="text-secondary">
+                        @LoadingText
+                    </p>
+                    }
+                    else
+                    {
+                    <p>
+                        Loading&hellip;
+                    </p>
+                    }
+
+            </Placeholder>
+        </Virtualize>}
+    else
+    {
+        @foreach (var item in Items ?? Array.Empty<TItem>())
```

Comment:
The "Items" property should not be null, so this null check operator (??) is redundant